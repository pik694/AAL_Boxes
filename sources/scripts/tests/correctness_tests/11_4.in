11
73 64 94
87 7 64
93 8 23
19 44 72
1 70 42
100 52 46
25 57 96
59 51 23
40 88 80
60 93 39
25 31 95

