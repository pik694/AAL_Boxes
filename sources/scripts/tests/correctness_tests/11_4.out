4 1 8 2 10 5 3 6 7 9 0 
1576440->1096768
